sass/_themes.scss

```diff
@@ -26,11 +26,13 @@ $COLOR_DARK_BLUE_2  : #101320;
 $COLOR_DARK_BLUE_3  : #191c31;
 $COLOR_DARK_BLUE_4  : #202641;
 
+$COLOR_LIGHT_BLACK_4_SOLID: #f4f4f6;
 $COLOR_LIGHT_BLACK : #5c5c5c;
 $COLOR_BRANDBLUE_40: #aaacba;
 $COLOR_BRANDBLUE_95: #353b5b;
 $COLOR_BRANDBLUE_85: #4a4f6c;
 $COLOR_BRANDBLUE_75: #60647e;
+$COLOR_DARK_BLUE_8 : rgba(127, 131, 151, 0.3);
```

Comment:
@shayanbinary please see my comment in FE channel regarding colours

Reply:
@ashkanx Sure, I just contact UI team about this.

Reply:
@ashkanx it's [here](https://app.zeplin.io/project/5c2deda538858f484127ada5/styleguide) in the dark theme pallet